drivers/hmis/lib/tasks/one_time_migration_20231121.rake

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+desc 'One time data migration from app.versions to warehouse.versions'
+# rails driver:hmis:migrate_versions_20231121
+task migrate_versions_20231121: [:environment] do
+  import_scope = GrPaperTrail::Version.where(item_type: 'Hmis::Hud::Client')
```

Comment:
```suggestion
  hmis_client_record_types = [
    'Hmis::Hud::Client',
    'Hmis::Hud::CustomClientName',
    'Hmis::Hud::CustomClientAddress',
    'Hmis::Hud::CustomClientContactPoint',
  ]
  import_scope = GrPaperTrail::Version.where(item_type: hmis_client_record_types)
```

I forgot about these, we want to bring them over too. I made this change and was able to run it locally (with the below modification)